uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Mail/Template/LeaveRequestNotificationTemplateTest.php

```diff
@@ -80,6 +82,7 @@ public function testGetTemplateParametersReturnsTheExpectedParametersForTheTempl
     $this->assertEquals($tplParams['leaveStatus'], $leaveRequestStatuses[$leaveRequest->status_id]);
     $this->assertEquals($tplParams['currentDateTime'], $dateTimeNow, '', 10);
     $this->assertEquals($tplParams['absenceTypeName'], $absenceType->title);
+    $this->assertEquals($tplParams['calculationUnitName'], $calculationUnitNames[$absenceType->calculation_unit]);
```

Comment:
We probably need to add this assertion to the TOIL and Sickness template tests

Reply:
Done.